ui/apps/everest/.e2e/release/init-deploy.e2e.ts

```diff
@@ -0,0 +1,180 @@
+// everest
+// Copyright (C) 2023 Percona LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+// http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+import { expect, test } from '@playwright/test';
+import {
+  deleteDbCluster,
+  suspendDbCluster,
+  resumeDbCluster,
+  restartDbCluster,
+} from '../utils/db-clusters-list';
+import { getTokenFromLocalStorage } from '../utils/localStorage';
+import { getClusterDetailedInfo } from '../utils/storage-class';
+import { resourcesStepCheck } from '../db-cluster/db-wizard/create-db-cluster/steps/resources-step';
+import {
+  moveForward,
+  submitWizard,
+  populateBasicInformation,
+} from '../utils/db-wizard';
+import { EVEREST_CI_NAMESPACES } from '../constants';
+import { waitForStatus, waitForDelete } from '../utils/table';
+
+test.describe.configure({ retries: 0 });
+
+[
+  { db: 'psmdb', size: 1 },
+  { db: 'psmdb', size: 3 },
+  { db: 'pxc', size: 1 },
+  { db: 'pxc', size: 3 },
+  { db: 'postgresql', size: 1 },
+  { db: 'postgresql', size: 3 },
+].forEach(({ db, size }) => {
```

Comment:
I like the idea of table tests, it's concise and observable. 
Let me to bring one thing to consideration. If the test fails, is there a way to quickly identify which one has failed? Maybe we could add definitive names (like `pxc1n`, `pxc3n`) which will be shown somehow during execution so when the test fails it won't be a problem to identify which case has failed.

Reply:
This is a good idea and I will see to add some identifier, initially my idea was if the test will be parametrized that the parameters need to be visible in test name like `test(`Cluster creation with ${db} and size ${size}` so if it fails you will see which combination has failed.
But in general I don't think that the structure with this parameters will stay because if one combination fails to re-run just that combination you need to remove other combinations from the test parameters and since we will run this in github action we need to be able to select which one to re-run.
But for now it is convenient to run it like this locally and try all combinations.

Reply:
Just for reference this is how it looks now:
```
Running 35 tests using 1 worker

  ✓  1 [auth] › ../auth.setup.ts:20:1 › Login (5.4s)
  ✓  2 [setup] › ../global.setup.ts:57:1 › Backup storages (1.8s)
  ✓  3 [setup] › ../global.setup.ts:120:1 › Close modal permanently (6.4s)
  ✓  4 [Chrome Stable] › ../release/init-deploy.e2e.ts:55:5 › Init deploy workflow › Cluster creation with psmdb and size 1 (1.2m)
  ✓  5 [Chrome Stable] › ../release/init-deploy.e2e.ts:121:5 › Init deploy workflow › Suspend cluster with psmdb and size 1 (9.9s)
  ✓  6 [Chrome Stable] › ../release/init-deploy.e2e.ts:131:5 › Init deploy workflow › Resume cluster with psmdb and size 1 (35.8s)
  ✓  7 [Chrome Stable] › ../release/init-deploy.e2e.ts:139:5 › Init deploy workflow › Restart cluster with psmdb and size 1 (31.0s)
  ✓  8 [Chrome Stable] › ../release/init-deploy.e2e.ts:150:5 › Init deploy workflow › Delete cluster with psmdb and size 1 (10.1s)
  ✓  9 [Chrome Stable] › ../release/init-deploy.e2e.ts:55:5 › Init deploy workflow › Cluster creation with psmdb and size 3 (1.9m)
  ✓  10 [Chrome Stable] › ../release/init-deploy.e2e.ts:121:5 › Init deploy workflow › Suspend cluster with psmdb and size 3 (25.1s)
  ✓  11 [Chrome Stable] › ../release/init-deploy.e2e.ts:131:5 › Init deploy workflow › Resume cluster with psmdb and size 3 (1.1m)
  ✓  12 [Chrome Stable] › ../release/init-deploy.e2e.ts:139:5 › Init deploy workflow › Restart cluster with psmdb and size 3 (1.5m)
  ✓  13 [Chrome Stable] › ../release/init-deploy.e2e.ts:150:5 › Init deploy workflow › Delete cluster with psmdb and size 3 (11.1s)
  ✓  14 [Chrome Stable] › ../release/init-deploy.e2e.ts:55:5 › Init deploy workflow › Cluster creation with pxc and size 1 (1.5m)
  ✓  15 [Chrome Stable] › ../release/init-deploy.e2e.ts:121:5 › Init deploy workflow › Suspend cluster with pxc and size 1 (25.1s)
  ✓  16 [Chrome Stable] › ../release/init-deploy.e2e.ts:131:5 › Init deploy workflow › Resume cluster with pxc and size 1 (56.2s)
  ✓  17 [Chrome Stable] › ../release/init-deploy.e2e.ts:139:5 › Init deploy workflow › Restart cluster with pxc and size 1 (1.1m)
  ✓  18 [Chrome Stable] › ../release/init-deploy.e2e.ts:150:5 › Init deploy workflow › Delete cluster with pxc and size 1 (10.5s)
  ✓  19 [Chrome Stable] › ../release/init-deploy.e2e.ts:55:5 › Init deploy workflow › Cluster creation with pxc and size 3 (3.9m)
  ✓  20 [Chrome Stable] › ../release/init-deploy.e2e.ts:121:5 › Init deploy workflow › Suspend cluster with pxc and size 3 (56.3s)
  ✓  21 [Chrome Stable] › ../release/init-deploy.e2e.ts:131:5 › Init deploy workflow › Resume cluster with pxc and size 3 (2.1m)
  ✓  22 [Chrome Stable] › ../release/init-deploy.e2e.ts:139:5 › Init deploy workflow › Restart cluster with pxc and size 3 (2.7m)
  ✓  23 [Chrome Stable] › ../release/init-deploy.e2e.ts:150:5 › Init deploy workflow › Delete cluster with pxc and size 3 (11.2s)
  ✓  24 [Chrome Stable] › ../release/init-deploy.e2e.ts:55:5 › Init deploy workflow › Cluster creation with postgresql and size 1 (1.0m)
  ✓  25 [Chrome Stable] › ../release/init-deploy.e2e.ts:121:5 › Init deploy workflow › Suspend cluster with postgresql and size 1 (29.7s)
  ✓  26 [Chrome Stable] › ../release/init-deploy.e2e.ts:131:5 › Init deploy workflow › Resume cluster with postgresql and size 1 (51.7s)
  ✓  27 [Chrome Stable] › ../release/init-deploy.e2e.ts:139:5 › Init deploy workflow › Restart cluster with postgresql and size 1 (31.0s)
  ✓  28 [Chrome Stable] › ../release/init-deploy.e2e.ts:150:5 › Init deploy workflow › Delete cluster with postgresql and size 1 (10.0s)
  ✓  29 [Chrome Stable] › ../release/init-deploy.e2e.ts:55:5 › Init deploy workflow › Cluster creation with postgresql and size 3 (1.3m)
  ✓  30 [Chrome Stable] › ../release/init-deploy.e2e.ts:121:5 › Init deploy workflow › Suspend cluster with postgresql and size 3 (10.0s)
  ✓  31 [Chrome Stable] › ../release/init-deploy.e2e.ts:131:5 › Init deploy workflow › Resume cluster with postgresql and size 3 (26.7s)
  ✓  32 [Chrome Stable] › ../release/init-deploy.e2e.ts:139:5 › Init deploy workflow › Restart cluster with postgresql and size 3 (1.3m)
  ✓  33 [Chrome Stable] › ../release/init-deploy.e2e.ts:150:5 › Init deploy workflow › Delete cluster with postgresql and size 3 (11.3s)
  ✓  34 [teardown] › ../global.teardown.ts:21:3 › Teardown › Delete backup storage (2.5s)
  ✓  35 [teardown] › ../global.teardown.ts:54:3 › Teardown › Logout (4.6s)
::warning title=Slow Test,file=[Chrome Stable] › ../release/init-deploy.e2e.ts::[Chrome Stable] › ../release/init-deploy.e2e.ts took 28.4m
::notice title=🎭 Playwright Run Summary::  35 passed (29.5m)
```

Reply:
In that case I think the output looks clear enough, my main concern was about how to identify which test has failed. I've missed that it's going to print the messages